tests/test_discounts.py

```diff
@@ -229,3 +234,50 @@ def test_sale_active(current_date, start_date, end_date, is_active):
         end_date=end_date)
     sale_is_active = Sale.objects.active(date=current_date).exists()
     assert is_active == sale_is_active
+
+
+@pytest.mark.parametrize('voucher_type',[
+    'collection', 'category', 'product', 'value', 'shipping'
+])
+def test_voucher_form_min_amount_spent(voucher_type):
+    data = {
+            'code' : 'C700C268D0DE',
+            'collection-min_amount_spent': 800,
+            'category-min_amount_spent': 800,
+            'product-min_amount_spent': 800,
+            'value-min_amount_spent': 800,
+            'shipping-min_amount_spent': 800,
```

Comment:
Only pass the correct data for the test case. Otherwise you won't know if the test passed because it's a side effect.

Instead, do:
```python
data[f'{voucher_type}-min_amount_spent'] = 800
```

Reply:
Even better... pass wrong data for the other ones (e.g. 400 instead of 800), so we ensure it ignores them 👍